### PR TITLE
docs: added macos installation instructions

### PR DESCRIPTION
**Pull request by deekay2310**
Signed-off-by: Dev Kumar <devskumar99@gmail.com>

### CODE REVIEW

**Comment by deekay2310** [comment]
Screen Shot 2022-06-28 at 1 48 00 PM
Screen Shot 2022-06-28 at 1 48 15 PM
Screen Shot 2022-06-28 at 1 48 22 PM

**Comment by benoitf** [comment]
Hello, for macOS we also provide .dmg files so maybe installation with brew is only a specific section ?

**Comment by deekay2310** [comment]
I mentioned at the end of the post that if Podman is already there, you can download the .dmg based on the architecture.

**Comment by deekay2310** [comment]
I can maybe move it to the beginning and specify that.

**Comment by deekay2310** [comment]
Screen Shot 2022-06-28 at 3 04 43 PM

File: website/docs/Installation/macos-install.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 3
+---
+
+# MacOS
+## Installing Podman Desktop on MacOS using .dmg file
+
+If you already have a Podman machine up and running, you can check out the [Downloads](/downloads/macos) section of this website to get the applicable .dmg file depending on your Mac Hardware Architecture (i.e. Intel or Apple M1).
```
Comment (benoitf) [inline website/docs/Installation/macos-install.md:8]: you only need podman being installed ( at least v4) but no need for a running machine. You can create it from Desktop

Also there is the Universal binary that is easier as you don't need to know your arch

File: website/docs/Installation/macos-install.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+Simply download the file from the [Downloads](/downloads/macos) section and open it in your machine to install Podman Desktop.
+
```
Comment (benoitf) [inline website/docs/Installation/macos-install.md:11]: I would avoid `Machine` here to not confuse people with podman machine

**Review by benoitf — APPROVED** [review]
overall Ok 👍 but if you can address comments

File: website/docs/Installation/macos-install.md (hunk 1 above)
Comment (deekay2310) [reply website/docs/Installation/macos-install.md:8]: so i should just delete the step 2?

File: website/docs/Installation/macos-install.md (hunk 2 above)
Comment (deekay2310) [reply website/docs/Installation/macos-install.md:11]: alright, will change it in the next push

**Pull request merged**